### PR TITLE
fix: image reference

### PR DESCRIPTION
**Pull request by PlateArmourProgrammer**
### Please check if the PR fulfills these requirements

- [x] The commit message follows our [guidelines](https://github.com/oasis-engine/engine/blob/main/.github/COMMIT_MESSAGE_CONVENTION.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)
fix image elements won't be garbage collected after onload callback

### CODE REVIEW

File: packages/core/src/asset/request.ts [hunk 1]
```diff
@@ -76,6 +76,10 @@ function requestImage<T>(url: string, config: RequestConfig): AssetPromise<T> {
         requestAnimationFrame(() => {
           //@ts-ignore
           resolve(img);
+          img.onload = null;
+          img.onerror = null;
+          img.onabort = null;
+          img = null;
```
Comment (GuoLei1990) [inline packages/core/src/asset/request.ts:82]: `img = null;` may not have any effect, after the function is executed, it will be automatically released。

Comment (PlateArmourProgrammer) [reply packages/core/src/asset/request.ts]: I did some tests. `img = null` is really useless.

**Review by GuoLei1990 — APPROVED** [review]
+1

**Review by gz65555 — APPROVED** [review]
+1

**Pull request merged**